console/src/components/Services/Events/CronTriggers/PendingEvents/PendingEvents.tsx

```diff
@@ -19,6 +21,15 @@ const PendingEvents: React.FC<Props> = props => {
   const { dispatch, currentTrigger } = props;
   const triggerName = currentTrigger ? currentTrigger.name : '';
 
+  const onCancelCronTrigger = (id: string, onSuccess: () => void) => {
+    const shouldCancelEvent = getConfirmation(
+      `You want to cancel the event ${id} of ${triggerName} cron trigger`
```

Comment:
is the `id` required here? Should I display the time the task is scheduled at instead?

Reply:
Changed it to show the time at which the event is scheduled.